tasks/sphinx.py

```diff
@@ -204,7 +204,7 @@ def _boundaries_tree(self, section_id, subsection_id):
                     FIRST(c.aggregate),
                     JSONB_Object_Agg(t.type || '/' || t.id, t.name),
                     'name' suggested_name,
-                    ARRAY_AGG(DISTINCT tab.timespan) timespan,
+                    STRING_AGG(DISTINCT tab.timespan, ',') timespan,
```

Comment:
Uhm, what happens when there are multiple timespans? This + slugify will probably do something ugly for US where there are multiple timespans, no? (I'm imagining `2015_2010_2015_2015_2010_2010` or something along those lines)

Reply:
Let me take a look